### PR TITLE
Convert color page to mdx

### PR DESCRIPTION
**Pull request by halocline**
[Deploy Preview](https://deploy-preview-2029--keen-mayer-a86c8b.netlify.app/foundation/color)

#### What does this PR do?

Converts legacy implementation of Color page from .js to .mdx.

#### Where should the reviewer start?

/pages/color.mdx

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [x] Small, medium, and large screen sizes
- [x] Cross-browsers (FireFox, Chrome, and Safari)
- [x] Light & dark modes
- [x] All hyperlinks route properly

**Accessibility Checks**
- [x] Keyboard interactions
- [x] Screen reader experience
- [x] Run WAVE accessibility plugin (Chrome) NOTE: There are a couple legacy contrast issues we may want to address

**Code Quality Checks**
- [x] Console is free of warnings and errors
- [x] Passes E2E commit checks
- [x] Visual snapshots are reasonable

### CODE REVIEW

File: aries-site/src/pages/foundation/color.mdx [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+If a font size is below 18px, AAA compliance will fail with
+default font weight on an HPE `green` <ColorSwatch background={{color: "green", dark: false}} /> background. In this case,
+apply `weight="bold"` to meet AAA compliance.
+
+<LightModeContrast />
+
+### Dark Mode
+
+AA compliance is met for text on a dark mode `green` <ColorSwatch background={{color: "green", dark: true}} /> background
+when using `text-strong` <ColorSwatch background={{color: "text-strong", dark: true}} /> color. To meet AAA compliance, use text
```
Comment (taysea) [inline aries-site/src/pages/foundation/color.mdx:122]: got some reason we're not getting correct code highlighting on `text-strong`

File: aries-site/src/pages/foundation/color.mdx [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+  LightPalette,
+  LightModeContrast,
+  Overlay,
+  StatusDark,
+  StatusLight,
+  TextDark,
+  TextLight
+} from '../../examples';
+
+You will notice that the HPE Design System Color guidance is
```
Comment (taysea) [inline aries-site/src/pages/foundation/color.mdx:34]: Don't think it matters in terms of how it renders, but noticing this has line breaks within paragraphs in the mdx as opposed to other files that just have on long line. Do we want to make things one line, or was this intentional for readability?

File: aries-site/src/pages/foundation/color.mdx [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+
+### Brand Color
+            
+HPE green symbolizes growth and vitality, commands attention and
+inspires action. Our green is an essential part of our brand
+identity and should be evident in every communication wherever
+possible. Always use the custom color specifications seen here
+to maintain consistency across channels and media.
+
+<BrandColor />
```
Comment (taysea) [inline aries-site/src/pages/foundation/color.mdx:65]: So much cleaner to review the page with the example extracted : )

File: aries-site/src/examples/foundation/color/ColorPalettes.js [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+        key={color.name}
+        textColor={textColor}
+        textSize={textSize}
+      />
+    ))}
+  </Box>
+);
+
+export const BrandColor = () =>
+  primaryColors && generateColorExamples(primaryColors, 'text-strong', 'large');
```
Comment (taysea) [inline aries-site/src/examples/foundation/color/ColorPalettes.js:40]: any reason in particular to make this larger?

**Review by taysea — COMMENTED** [review]
Minor comments but nothing that should necessarily block moving forward with the PR.

**Review by taysea — APPROVED** [review]

File: aries-site/src/pages/foundation/color.mdx (hunk 2 above)
Comment (halocline) [reply aries-site/src/pages/foundation/color.mdx:34]: I find it easier for readability. The long lines tend to bother me, but just a preference.

File: aries-site/src/pages/foundation/color.mdx (hunk 3 above)
Comment (halocline) [reply aries-site/src/pages/foundation/color.mdx:65]: totally

File: aries-site/src/components/content/ColorSwatch.js [hunk 5]
```diff
@@ -12,6 +12,7 @@ export const ColorSwatch = ({ background }) => {
 
   return (
     <Box
+      as="span"
```
Comment (britt6612) [inline aries-site/src/components/content/ColorSwatch.js:15]: 👍

File: aries-site/src/examples/foundation/color/ColorPalettes.js (hunk 4 above)
Comment (halocline) [reply aries-site/src/examples/foundation/color/ColorPalettes.js:40]: Contrast, thus readability is poor at "small" font size. Was waffling whether we should leave alone or change.

**Review by britt6612 — APPROVED** [review]

File: aries-site/src/pages/foundation/color.mdx (hunk 1 above)
Comment (halocline) [reply aries-site/src/pages/foundation/color.mdx:122]: Yeah it was being real fussy. Not sure what is going on there. Same thing is implemented elsewhere and renders just fine.

**Pull request merged**